openassessment/xblock/schema.py

```diff
@@ -121,6 +121,7 @@ def datetime_validator(value):
             'required': bool,
             'must_grade': All(int, Range(min=0)),
             'must_be_graded_by': All(int, Range(min=0)),
+            Required('enable_flexible_grading', default=True): bool,
```

Comment:
It should default to false, since that is the current behaviour.

Reply:
I'd agree with you and keep the existing behavior, but the ticket description says "By default for problems we’d like to have this be enabled but for existing problems this might be disabled.", which led me to switch to true.

Reply:
Alright, in that case, we need to make sure that any existing blocks that don't have this field are handled properly, i.e. if this field is undefined, it should default to false. 

So I think it can default to false in the schema, but default to true for the data used to fill in the default values in `defaults.py`.